src/main/java/uk/gov/companieshouse/presenter/account/service/PresenterAccountService.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.presenter.account.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.presenter.account.model.PresenterAccountAddress;
+import uk.gov.companieshouse.presenter.account.model.PresenterAccountDetails;
+import uk.gov.companieshouse.presenter.account.model.PresenterAccountName;
+import uk.gov.companieshouse.presenter.account.model.request.presenter.account.PresenterRequest;
+
+@Service
+public class PresenterAccountService {
+    
+    private Logger logger;
+
+    @Autowired
+    public PresenterAccountService(Logger logger) {
+        this.logger = logger;
+    }
+
+    public String createPresenterAccount(PresenterRequest presenterRequest){
+        PresenterAccountDetails presenterDetails = new PresenterAccountDetails(
```

Comment:
I think a mapper class would be better used for creating the model from the request. I don't like that the model object is coupled to the request object. A mapper would do the setting for each individual field, as opposed to the static functions you've created in the models.